tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -6,10 +6,13 @@ Spatial shaders
 Spatial shaders are used for shading 3D objects. They are the most complex type of shader Godot offers.
 Spatial shaders are highly configurable with different render modes and different rendering options
 (e.g. Subsurface Scattering, Transmission, Ambient Occlusion, Rim lighting etc). Users can optionally
-write vertex, fragment, and light processor functions to affect how objects are drawn.
+write vertex, fragment, and light processor functions to affect how objects are drawn. 
+
+
 
 Render modes
 ^^^^^^^^^^^^
+For visual examples of these render modes, see :ref:`Standard Material 3D and ORM Material 3D<doc_standard_material_3d>`.
```

Comment:
Would like suggestions on the best wording for this. We do need to link to the Standard Material page, since it's the best/only documentation for many features of spatial shaders. I had it in the opening paragraph for a while. Putting the link here keeps it at the top of the page, while being directly relevant to the section.

Reply:
Seems okay!

Reply:
Yep, seems fine